lib/controller/controller.go

```diff
@@ -1055,12 +1059,35 @@ func (ctrl *ProvisionController) provisionClaimOperation(claim *v1.PersistentVol
 		return err
 	}
 
+	var selectedNode *v1.Node
+	var allowedTopologies []v1.TopologySelectorTerm
+	if ctrl.kubeVersion.AtLeast(utilversion.MustParseSemantic("v1.11.0")) {
+		// Get SelectedNode
+		if nodeName, ok := claim.Annotations[annSelectedNode]; ok {
+			selectedNode, err = ctrl.client.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{}) // TODO (verult) cache Nodes
+			if err != nil {
+				strerr := fmt.Sprintf("Failed to get target node: %v", err)
+				glog.Infof("unexpected error getting target node %q for claim %q: %v", nodeName, claimToClaimKey(claim), err)
```

Comment:
Do we need to log the error if it's being returned?

Reply:
Same as the other comment

Reply:
@wongma7 what do you think of potentially cleaning this up (as a separate PR)?  I think the style of logging an error that you also return is unusual

Reply:
yes I will clean it up on next week, the caller of `provisionClaimOperation` should log the error.

upstream is weirder because the error is bubbled up through goroutinemap and such so I imagine there is a logical explanation for the weirdness but at least here we should make it simpler if we can

Reply:
ACK